### ISSUE TITLE
Using five or more STUN/TURN servers causes problems

### ISSUE THREAD

**Issue by whitesnow9291** [stephenlb/webrtc-sdk#76]
Hello :)
I followed tutorial for broadcasting and viewer. (I exactly used same code and only changed keys)
viewer don't show broadcaster's cam video.
And I got this error. `Using five or more STUN/TURN servers causes problems`
And this is error messages from console.

![Screen Shot 2020-04-30 at 2 50 39 PM](https://user-images.githubusercontent.com/25058760/80680806-0a6f3380-8af2-11ea-85b2-9f31091543b9.png)

Is this because keys are of free version? or is this issue?

Thanks

**Comment by stephenlb** [comment]
Good catch!

**Comment by stephenlb** [comment]
This may be related to other issues faced.  
https://github.com/stephenlb/webrtc-sdk/blob/master/js/webrtc-v2.js#L48-L72

Will update...

**Comment by stephenlb** [comment]
Did a quick fix: https://github.com/stephenlb/webrtc-sdk/commit/bd71202dc5e0e38618cbf9e1d072ac7690f951b8

**Comment by whitesnow9291** [comment]
Yes, it's now fixed.
Btw, I have one more issue.
viewer don't show broadcasters cam.
Please check here.
[broadcast](https://disney.lat/call/broadcast.html)
[viewer](https://disney.lat/call/view.html)
It's just stuck after showing this log `Fetching broadcaster's video`
Thanks

**Comment by stephenlb** [comment]
that last one was due to the broadcaster not streaming before the viewer starts.  added a notification system when the broadcaster starts so the viewer can reload.  

It's not perfect, its a start though.  https://github.com/stephenlb/webrtc-sdk/commit/89da1078993d527ca954dd662365a6ea3bea6371

**Comment by whitesnow9291** [comment]
Hi @stephenlb 
Thanks but unfortunately it doesn't work for me.
I have cloned and deployed without change.
I have tested for both chrome and firefox. (both not working)
Also I have tested with my own pub,sub key pairs for both browsers (both not working)
I have attached screens shots for each browser.
1. Chrome
- broadcaster.html screenshot
![2020-05-02 21_48_03-Microsoft Edge](https://user-images.githubusercontent.com/25058760/80866107-8d91b480-8cbf-11ea-92d9-2af0230b2675.png)
It seems to work, as it doesn't show any error in console.
- viewer.html screenshot
![Screen Shot 2020-05-02 at 9 44 15 PM](https://user-images.githubusercontent.com/25058760/80866122-a8fcbf80-8cbf-11ea-8957-ae56cbb65632.png)
It has no error message, but it doesn't show any cam in screen. so it's not working
2. Firefox
- broadcaster.html screenshot
![2020-05-02 21_50_34-Microsoft Edge](https://user-images.githubusercontent.com/25058760/80866150-d77a9a80-8cbf-11ea-9e3b-9830cd66d09d.png)
Firefox shows errors, and it doesn't show cam also
- viewer.html screenshot
![Screen Shot 2020-05-02 at 9 44 38 PM](https://user-images.githubusercontent.com/25058760/80866179-eeb98800-8cbf-11ea-92d2-e3a37f0bd8a0.png)
viewer also shows errors and don't show any cam.

I didn't test on safari because it doesn't seem to be browser issue.
Maybe I need to find other webrtc framework or should use bare webrtc.
Thanks

**Comment by stephenlb** [comment]
Yes this is the start, and is an incomplete example. The app code needs work, as this is just an example.  If you get a chance to also check out https://xirsys.com/ they have a paid solution ready to go!

**Comment by stephenlb** [comment]
Should work in Firefox now.  It's not going to be exactly what you might be looking for.  However it's better.
